Migrations/Mysql/Version20220218143832.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Migrations\AbortMigrationException;
+
+/**
+ * Auto-generated Migration: Please modify to your needs! This block will be used as the migration description if getDescription() is not used.
+ */
+class Version20220218143832 extends AbstractMigration
+{
+
+    /**
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    /**
+     * @param Schema $schema
+     * @return void
+     * @throws AbortMigrationException
+     */
+    public function up(Schema $schema): void
+    {
+        // this up() migration is autogenerated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on "mysql".');
+        
+        $this->addSql('DROP INDEX sourceIdentifier_index ON sitegeist_csvpo_domain_translationoverride');
+        $this->addSql('CREATE INDEX sourceIdentifier_index ON sitegeist_csvpo_domain_translationoverride (sourceIdentifier(255))');
+    }
```

Comment:
Sow wie ich das sehe fasst die migration aber die db definition gar nicht an. Da stimmt etwas nicht.

Reply:
Oops, evtl. habe ich "kopieren" in den Github Web-Editor die falsche Migration erwischt 🙈 
Ich guck mir das gleich nochmals an ☝️

Reply:
Ich habe glaub wirklich das falsche Migration-File erwischt 🙈 
[Hier](https://github.com/sitegeist/Sitegeist.CsvPO/pull/12/commits/ea28b1e6bd721e68e2b63a72d7bf39a493b9d406) nun die korrekte Version.